internal/config/settings.go

```diff
@@ -295,7 +295,11 @@ func InitLocalSettings(settings map[string]interface{}, path string) {
 	for k, v := range parsedSettings {
 		if strings.HasPrefix(reflect.TypeOf(v).String(), "map") {
 			if strings.HasPrefix(k, "ft:") {
-				if settings["filetype"].(string) == k[3:] {
+				filetype, ok := settings["filetype"]
+				if !ok {
+					continue
+				}
+				if filetype.(string) == k[3:] {
```

Comment:
This could be just `if filetype, ok := settings["filetype"]; ok && filetype.(string) == k[3:] {` ?

And about the commit message: as always, it would be good if it described not just _what_ the commit does but _why_ does it do what it does (i.e. in this case, _why_ does `InitLocalSettings()` need to check for presence of local `filetype`). So that readers (including yourself after a month) don't need to guess that or dig through discussions in this PR.

Reply:
> So that readers (including yourself after a month) don't need to guess [...]

Yes, you're right, I'm forgetful. I'm sorry.